packages/switch/src/react/index.js

```diff
@@ -116,6 +117,15 @@ class Switch extends React.Component {
   }
 }
 
+Switch.defaultProps = {
```

Comment:
doesn't matter a bunch to me, but `defaultProps` have (probably) always come after `propTypes` in these components, at least the ones I just spot-checked.

Reply:
👍 i'll move